pkg/controllers/node/node_syncer.go

```diff
@@ -45,45 +45,61 @@ func (c *NodeController) OnUpdates(updates []bapi.Update) {
 		switch upd.UpdateType {
 		case bapi.UpdateTypeKVNew:
 			fallthrough
+
 		case bapi.UpdateTypeKVUpdated:
 			n := upd.KVPair.Value.(*apiv3.Node)
```

Comment:
We might want to consider having the syncer also send updates when HEPs are modified. That way if a user tweaks an auto-HEP we can correct it.

I suggest holding off on doing that for now and we can consider it as an optional add-on later on. Don't want to complicate things just yet, but thought it worth mentioning.